src/containers/AppStore/containers/AppDetails/AppDetailsSlider/Styles.ts

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import {Container as SharedContainer} from 'components';
+
+export const Container = styled.div`
+  background: rgba(246, 249, 252, 0.5);
```

Comment:
The carousel background is white in Figma

Reply:
ah yes, dk why i set that color, maybe designer changed it